acos_client/v30/file/ssl_cert.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2015,  Tobit Raff,  A10 Networks.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import acos_client.errors as acos_errors
+import acos_client.v30.base as base
+#from acos_client import multipart
+
+
+class SSLCert(base.BaseV30):
+
+    url_prefix = '/file/ssl-cert/'
+
+    def get(self, file):
+        return self._get(self.url_prefix + file)
+
+    def exists(self, file):
+        try:
+            self.get(file)
+            return True
+        except acos_errors.NotFound:
+            return False
+
+    def _set(self,
+             file=None,
+             cert=None,
+             size=None,
+             certificate_type=None,
+             pfx_password=None,
+             csr_generate=None,
+             action=None,
+             dst_file=None,
+             update=False,
+             **kwargs):
+
```

Comment:
Oops, yeah I meant to not format it this way. It's ugly. I was just using it while I was deciding what arguments I wanted to use. Easier to delete a line than a chunk of text.